Classes/JTLocalizeUtils.m

```diff
@@ -97,7 +97,6 @@ + (void)setLocalizationBundleToPath:(NSString *)bundlePath stringsTableName:(NSS
     // directly to the sub-path of the give locale (overriding the OS-based lookup)
     NSString *localePath = [bundle pathForResource:preferredLocale ofType:@"lproj"];
     if (localePath != nil) {
-        [self instance].stringsTableName = nil;
```

Comment:
This allows us to use a different tableName even when reading the localization from the bundle, which is set on line#83 (When we set `nil` it used `Localizable` by default)

Reply:
Looks good.